scene/gui/option_button.cpp

```diff
@@ -234,6 +236,10 @@ Ref<Texture2D> OptionButton::get_item_icon(int p_idx) const {
 }
 
 int OptionButton::get_item_id(int p_idx) const {
+  if (p_idx == NONE_SELECTED) {
+    return NONE_SELECTED;
+  }
```

Comment:
Note that you should use tabs for indentation.

Reply:
And after/while fixing that, please squash commits together by doing a rebase and force push (see [PR workflow](https://docs.godotengine.org/en/latest/community/contributing/pr_workflow.html#the-interactive-rebase)).

Reply:
fixed and squashed!